app/components/ArrowKeyNavigation.tsx

```diff
@@ -0,0 +1,51 @@
+import { observer } from "mobx-react";
+import * as React from "react";
+import {
+  useCompositeState,
+  Composite,
+  CompositeStateReturn,
+} from "reakit/Composite";
+
+type Props = {
+  children: (composite: CompositeStateReturn) => React.ReactNode;
+  onEscape?: (ev: React.KeyboardEvent<HTMLDivElement>) => void;
```

Comment:
```suggestion
  onExit?: (ev: React.KeyboardEvent<HTMLDivElement>) => void;
```

Maybe? "escape" is just mapped so thoroughly to the key in my head

Reply:
Sure, not a blocker just an opinion

Reply:
yeah I prefer escape for now